build.js

```diff
@@ -102,6 +105,7 @@ class WebComponentBuild {
 
 	__vervangGovFlandersImportsDoorMinifiedImports(file) {
 		replace(`'/node_modules/@govflanders/(.*).js'`, `'/node_modules/@govflanders/\\$1.min.js'`, file);
+		replace(`'/node_modules/@govflanders/(.*).min.min.js'`, `'/node_modules/@govflanders/\\$1.min.js'`, file);
```

Comment:
@pascalriquier bij deze kwam er soms een min.min.js versie als er al een min versie werd genomen. Ik heb nu deze quick fix er in gezet om de testen te doen slagen, maar misschien is er een properdere manier?

Reply:
wij hebben dat genoteerd in hetvolgende ticket om overal in de src code voor de @govflanders stuff NIET de min versie te gebruiken, maar deze dan ook in de minified versie wel te gebruiken, dus dit lijkt me niet nodig op die manier.